app/controllers/questions_controller.rb

```diff
@@ -1,5 +1,6 @@
 class QuestionsController < ApplicationController
   before_action :assign_question, only: %i(show edit update destroy)
+  before_action :pluck_name_id_documentation, only: %i(new edit)
```

Comment:
Same as above: a better naming could improve the comprehension.
Better it's an "action", starting the method name by an "action verb" would help a lot: `assign_documentations_name_id`.